test-suite/tests/compact-manifest.jsonld

```diff
@@ -586,6 +586,14 @@
       "context": "compact-0072-context.jsonld",
       "expect": "compact-0072-out.jsonld"
     }, {
+      "@id": "#t0073",
+      "@type": ["jld:PositiveEvaluationTest", "jld:CompactTest"],
+      "name": "Mapped @id and @type",
+      "purpose": "Ensure that compaction works with mapped @id and @type",
+      "input": "compact-0073-in.jsonld",
+      "context": "compact-0073-context.jsonld",
+      "expect": "compact-0073-out.jsonld"
+      }, {
```

Comment:
For this to work consistently, you need to add "option": {"base": "http://example/base/"} or similar to the test definition.